### PR TITLE
Fatal readme update

### PR DESCRIPTION
**Pull request by WhitShake**
Added information for Fatal Diagnostic changes to README

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/README.md [hunk 1]
```diff
@@ -54,3 +54,25 @@ You can rename a swagger file as mentioned below and upload it to APIView if you
 TypeSpec API revision is generated automatically from a pull request and this should be good enough in most scenarios. You can also generate API revision manually for a TypeSpec package by providing URL path to TypeSpec package specification root path.
 1. Click and `Create Review` and select TypeSpec from language dropdown.
 2. Provide URL to typespec project root path.
+
+
```
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/README.md:58]:
```suggestion
```

File: src/dotnet/APIView/APIViewWeb/README.md [hunk 2]
```diff
@@ -54,3 +54,25 @@ You can rename a swagger file as mentioned below and upload it to APIView if you
...
+## Fatal (Level 4) Diagnostic
+
+### Introduction
+
+APIView has introduced a new diagnostic level: Fatal (Level 4). This level captures the most critical issues requiring immediate intervention. Issues at this level are considered to be blocking and must be resolved or overridden before the API revision can be approved.
+
+### Usage Instructions
+
+1. Defining Fatal Diagnostics:
```
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/README.md:68]: Could you add an example of how it looks in the parser?

**Comment by maririos** [comment]
@praveenkuttappan this never got merged. Do you think we should still merge it considering the changes you are making?

**Comment by praveenkuttappan** [comment]
Token schema has the levels documented in new schema tsp. So I think we don't need this documented here. I can make sure to include additional info about the fatal level in it.

**Pull request closed without merging**